typings/sharify.d.ts

```diff
@@ -19,6 +19,7 @@ declare module "sharify" {
       readonly FORCE_CLOUDFRONT_URL: string
       readonly GEMINI_CLOUDFRONT_URL: string
       readonly METAPHYSICS_ENDPOINT: string
+      readonly NODE_ENV: string
```

Comment:
👍 Now the only thing missing is adding it to the `sharify` stub for development https://github.com/artsy/reaction/blob/8f6ac8b73b1c6cb8e79985a77ca86d3f6d06d342/.storybook/webpack.config.js#L31-L34